ha_ibmdb2i.cc

```diff
@@ -2791,10 +2791,9 @@ int ha_ibmdb2i::prepReadBuffer(ha_rows rowsToRead, const db2i_file* file, char i
     forceSingleRowRead = true;
     rowsToRead = 1;
   }
+  rowsToRead = std::min(stats.records+1,std::min(rowsToRead, static_cast<ha_rows>(DEFAULT_MAX_ROWS_TO_BUFFER))); 
```

Comment:
With the change of `DEFAULT_MAX_ROWS_TO_BUFFER` to an `ha_rows`, this static_cast should be unnecessary.

Reply:
Yes fixed this to remove the unnecessary cast 

https://github.com/zendtech/db2iengine/pull/23/files#diff-e0e69605b45f256c19b7588c8c2d3d9b30bed1f1cd54f6fd742807a4d51e61e3R2794